scheduler/src/cook/rest/api.clj

```diff
@@ -1190,13 +1191,13 @@
         (cond-> {:backfilled false ;; Backfill has been deprecated
                  :compute-cluster (fetch-compute-cluster-map db (:instance/compute-cluster instance))
                  :executor_id (:instance/executor-id instance)
-                 :hostname hostname
                  :ports (vec (sort (:instance/ports instance)))
                  :preempted (:instance/preempted? instance false)
-                 :agent_id (:instance/slave-id instance)
-                 :slave_id (:instance/slave-id instance)
                  :status (name (:instance/status instance))
                  :task_id task-id}
+          hostname (assoc :hostname hostname)
```

Comment:
Actually. Why not subsitute in the fake data in here, instead of in the database?

I.e., in the cond-> above, we unconditionally include the hostname, but with the code:

> `   :hostname (or hostname "not-binpacked-hostname")`

> The cs cli requires hostname, slave-id, and agent-id  to be set. For just-launched pods on k8s where we are letting k8s to the binpacking, they won't be available. Thus we substitute fake numbers temporarily.

With a TODO: to fix the CLI. 

This way we also don't need to store garbage data into the database. (And we can delete the code that does that.)

Reply:
I will add this as a follow-up.